forge/csrc/reportify/reportify.cpp

```diff
@@ -425,5 +458,29 @@ void dump_graph(
     std::string default_dir = get_default_reportify_path("");
     dump_graph(default_dir, test_name, graph_prefix, graph, report_path);
 }
+
+void dump_mlir(const std::string& name, mlir::ModuleOp* module)
+{
+    if (env_as<bool>("FORGE_DISABLE_REPORTIFY_DUMP"))
+        return;
+
+    std::string path = get_default_reportify_path("");
+    std::string report_path = get_mlir_reports_relative_directory();
+
+    std::string full_report_path = build_report_path(path, module->getName()->str(), report_path);
+
+    JsonNamePairs json_pairs = create_jsons_for_mlir(name, module);
+    json root_json = json_pairs.back().first;
+
```

Comment:
Same here, the second block is building json name or full json path. 

Not major requirement, mostly personal preferences for code readability :))